docs/programming-oak.md

```diff
@@ -223,6 +223,19 @@ Application Configuration File.
 
 Each of these steps is described in the following sections.
 
+### Compiling to WASM Module
+
+In order to build a WebAssembly module for an Oak WebAssembly Node, written in
+Rust, `cargo build` should be used with `--target=wasm32-unknown-unknown`, as
+follows:
+
+```bash
+cargo -Zunstable-options build --release \
+  --target=wasm32-unknown-unknown \
+  --manifest-path=examples/hello_world/module/rust/Cargo.toml \
+  --out-dir=examples/hello_world/bin
+```
```

Comment:
Maybe add something about "the `-Zunstable-options .. --out-dir` options ensure that the resulting binary is copied into a specific directory to allow later steps to find it"

Reply:
Thanks. Done.